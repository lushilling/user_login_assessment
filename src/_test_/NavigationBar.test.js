import React from 'react';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';

import NavigationBar from '../Components/NavigationBar';


it('renders App', () => {
    const tree = renderer.create(<NavigationBar />).toJSON()
    expect(tree).toMatchSnapshot();
})

it('renders app without crashing', () => {
    shallow(<NavigationBar />);
});