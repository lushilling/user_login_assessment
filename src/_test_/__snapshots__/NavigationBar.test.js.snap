// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders App 1`] = `
<div>
  <nav
    className="navbar navbar-expand-md navbar-dark bg-dark"
  >
    <a
      className="navbar-brand"
      href="/"
    >
      Assessment
    </a>
    <button
      className="navbar-toggler"
      onClick={[Function]}
      type="button"
    >
      <span
        className="navbar-toggler-icon"
      />
    </button>
    <div
      className="collapse navbar-collapse"
      style={Object {}}
    >
      <ul
        className="ml-auto navbar-nav"
      >
        <li
          className="nav-item"
        >
          <a
            className="nav-link"
            href="./Register"
            onClick={[Function]}
          >
            Register
          </a>
        </li>
        <li
          className="nav-item"
        >
          <a
            className="nav-link"
            href="./Login"
            onClick={[Function]}
          >
            Login
          </a>
        </li>
        <li
          className="nav-item"
        >
          <a
            className="nav-link"
            href="./UserList"
            onClick={[Function]}
          >
            User List
          </a>
        </li>
      </ul>
    </div>
  </nav>
</div>
`;
